harmony_browse_image_generator/sizes.py

```diff
@@ -125,19 +125,22 @@ def get_target_grid_parameters(message: Message, data_array: DataArray) -> GridP
 
     """
     target_crs = choose_target_crs(message.format.srs, data_array)
-    target_scale_extent = choose_scale_extent(message, target_crs)
+    target_scale_extent = choose_scale_extent(message, target_crs, data_array)
     target_dimensions = choose_target_dimensions(
         message, data_array, target_scale_extent, target_crs
     )
     return get_rasterio_parameters(target_crs, target_scale_extent, target_dimensions)
 
 
-def choose_scale_extent(message: Message, target_crs: CRS) -> ScaleExtent:
-    """Return the scaleExtent of the target image.
+def choose_scale_extent(
+    message: Message, target_crs: CRS, data_array: DataArray
+) -> ScaleExtent:
+    """Return the scaleExtent for the target image.
 
-    Check the message for a defined scale extent and returns that or returns
-    the best alternative based on the target_crs either returning it from a
-    lookup based on the ICD, or computed with pyproj.area_of_use
+    Returns a scale extent found in the input Message.
+
+    Otherwise, computes a bounding box in the target CRS based on the input
+    granule extent.
```

Comment:
I'm looking into this. We do already [require any input geotiff to explicitly have a crs](https://github.com/nasa/harmony-browse-image-generator//blob/mhs%2FDAS-2115%2Fdefault-to-better-scale-extents/harmony_browse_image_generator/browse.py#L335) when opened with rioxarray.

Reply:
I feel like there may be a degenerate case where this could fail, but I think the error would be explicit enough to tell you your data is bad.  I'm not sure what else we could do with that file though.